tests/FSharp.Compiler.ComponentTests/EmittedIL/ComputedCollections/ComputedCollections.fs

```diff
@@ -17,59 +18,59 @@ module ComputedCollections =
     [<Theory; Directory(__SOURCE_DIRECTORY__, Includes=[|"Int32RangeArrays.fs"|])>]
     let ``Int32RangeArrays_fs`` compilation =
         compilation
-        |> verifyCompilation
+        |> verifyCompilation false
```

Comment:
I am okay with this PR not turning realSig's default to true **yet**.

But when we do, I think we should be running as much tests as possible with the same default.
Will it be a painful update, or was there another reason for using "false" here?

Reply:
I can double the test, so it does both, although InlineData and Directory don't play nice together.